### ISSUE TITLE
Create topic based timeouts

### ISSUE THREAD

**Issue by yatharthranjan** [RADAR-base/RADAR-Backend#74]
Timeouts for monitor should be based on topics. Since for example, questionnaire app will be considered disconnected after a very long time compared to the passive app.

**Comment by nivemaham** [comment]
Right. Currently, a similar configuration used per source-type to decide DISCONNECTED state per source-type. If that would be relevant here as well, we should probably put these configurations in one place.
